### PR TITLE
Scaffold lead prediction page with empty placeholders 

### PR DESCRIPTION
**Pull request by justinebreuch**
## Description

Addresses: Create basic scorecard component with placeholder data

Scaffolds more of the lead prediction page. Adds three factors for lead score to the page. 

Screen Shot 2022-07-29 at 2 33 57 PM

### New

3 most important factors for lead prediction are on page now. Placeholders for images and text. 

## Testing and Reviewing

See image.

### CODE REVIEW

**Review by beekley — CHANGES_REQUESTED** [review]
LGTM, with small changes about typescript stuff.

File: client/src/assets/messages/scorecard_summary_messages.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
+ * Text to use on the scorecard summary page.
+ */
+export class ScorecardSummaryMessages {
+    static SCORECARD_SUMMARY_PANEL_HEADER = "What's in your score";
```
Comment (kailamjeter) [inline client/src/assets/messages/scorecard_summary_messages.js:5]: nit: single quotes to be consistent with other strings

File: client/src/assets/messages/scorecard_summary_messages.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    static SCORECARD_SUMMARY_PANEL_SUBHEADER = 'The most relevant data used to determine ' + 'your lead score';
```
Comment (kailamjeter) [inline client/src/assets/messages/scorecard_summary_messages.js:6]: nit: why is this concatenated?

**Review by kailamjeter — APPROVED** [review]

File: client/src/assets/messages/scorecard_summary_messages.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/**
```
[the remark this answers is not in the document]
Comment (justinebreuch) [reply client/src/assets/messages/scorecard_summary_messages.js]: No I had to manually remove

File: client/src/api/api_client.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
```
[the remark this answers is not in the document]
Comment (justinebreuch) [reply client/src/api/api_client.js]: Manually removed

File: client/src/assets/messages/scorecard_summary_messages.js (hunk 2 above)
Comment (justinebreuch) [reply client/src/assets/messages/scorecard_summary_messages.js]: Good eye - not sure. Undone

File: client/src/assets/messages/scorecard_summary_messages.js (hunk 1 above)
Comment (justinebreuch) [reply client/src/assets/messages/scorecard_summary_messages.js]: yeah good point

**Review by beekley — APPROVED** [review]

**Pull request merged**